src/hooks/useApi.ts

```diff
@@ -4,6 +4,7 @@ import { useAlert } from './useAlert';
 
 interface Options {
    authenticate?: boolean;
+   multipart?: boolean;
```

Comment:
제가 고려 못한 부분에 대한 좋은 업데이트 같습니다! 
contentType으로 작성해주시면 다른 유형에 대해서도 사용할 수 있을 것 같습니다!

```suggestion
   type contentType = 'multipart/formed-data' | 'Application/json' | 'Application/x-www-form-urlencode';
   contentType?: contentType
```